man/systemd.unit.xml

```diff
@@ -115,6 +115,14 @@
     <para>Unit files are loaded from a set of paths determined during
     compilation, described in the next section.</para>
 
+    <para>Unit files can be parameterized by a single argument called the
+    "instance name". The unit is then constructed based on a "template file"
+    which serves as the definition of multiple services or other units. The full
+    name is given by appending an "@" to the unit name, followed by the instance
```

Comment:
hmpff, the "unit name" I think includes the type suffix, hence appending "@" to that would result in "foobar.service@" which is borked...